src/MUIDataTable.js

```diff
@@ -328,6 +335,7 @@ class MUIDataTable extends React.Component {
       separator: ',',
     },
     setTableProps: () => ({}),
+    selectToolbarPlacement: 'replace',
```

Comment:
I think we should make the options a bit more consistent in their naming, so either `replace` and `above` or `replace-toolbar` and `above-toolbar`.